planner/core/plan_test.go

```diff
@@ -448,3 +449,18 @@ func (s *testPlanNormalize) TestDecodePlanPerformance(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(time.Since(start).Seconds(), Less, 3.0)
 }
+
+func (s *testPlanNormalize) TestJSONExplain(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+	tk.MustExec("DROP TABLE IF EXISTS t1, t2")
+	tk.MustExec("CREATE TABLE t1 (id INT NOT NULL PRIMARY KEY, b INT)")
+	tk.MustExec("CREATE TABLE t2 (id INT NOT NULL PRIMARY KEY, t1_id INT)")
+
+	var js core.JSONOperatorRow
+	result := tk.MustQuery("EXPLAIN FORMAT=JSON SELECT t1.* FROM t1 INNER JOIN t2 ON t1.id=t2.t1_id")
```

Comment:
Please add more cases to cover more operators.

Reply:
@qw4990 I was worried about test stability as statistics and operator auto-naming can change. What I arrived at, was to use a regular expression. PTAL again, thx  :-)